### PR TITLE
add xk6-plist to the list of extensions

### PR DESCRIPTION
**Pull request by mcosta74**

### CODE REVIEW

File: src/data/doc-extensions/extensions.json [hunk 1]
```diff
@@ -1111,6 +1111,21 @@
       "categories": ["Data"],
       "tiers": ["Community"],
       "cloudEnabled": false
+    },
+    {
+      "name": "xk6-plist",
+      "description": "Parse/serialize CVS payloads",
```
Comment (heitortsergent) [inline src/data/doc-extensions/extensions.json:1117]:
```suggestion
      "description": "Parse/serialize property list (.plist) payloads",
```

Comment (mcosta74) [reply src/data/doc-extensions/extensions.json:1117]: done, thanks for the catch

**Comment by mcosta74** [comment]
> @mcosta74, sorry, but another change to the registry is causing a merge conflict. Please rebase your branch and update the pull request. Thank you!

done!

**Review by javaducky — APPROVED** [review]
Thank you!

**Pull request merged**